20181213/MINUTES.md

```diff
@@ -0,0 +1,138 @@
+     - question: could an experimental channel be used simultaneouly by two
+     doors?. Zibi answers that it should not work, if it works is a bug.
+     It would need to change a lot of things for making this working.
+     For the tango zerod should not be problem, tango would serialize it
+     and read a tango attribute. But for others than zerod it will depend.
+     In general each element in Sardana should be only used at once.   
+     Gregorz wanted to read a Tango Attribute with the epoch time from
+     different Doors. Zibi suggests to create the epoch time from the
+     dt of the scan. The important thing is to save it in the file,
+     and dt and a start time as saved (but dt is not relative to the start
+     time), Zibi suggests to make it coherent and use then these
+     information and not add any extra ones.
+     - MacroServer requires qt bindings due to parsers in spock.
+     This issue is already solved with the change in spock syntax.
+     - Time between scan points is larger than the expected
+     one, Gregorz thinks that is related to the MG synchronitation.
+     There are some issues related to this, ex. by DESY. SEP18 could avoid
+     some delays because some preparation goes to the beginning of the scan.
+     It will be investigated with tools for time profiling and see if
+     there is some room for improvement.
+
+   * ALBA:
+     - nothing to report
+
+   * MAXIV:
+     - when running long scans the macroserver hangs, they are trying
+     to investigate it. It is not clear if it is a sardana issue, it could
+     be the write of files. They can not reproduce it and it happens in
+     one of the beamlines once or twice per week. It could be a network
+     problem. In the logs one can see that the acquisiton starts but no
+     events are sent any more. There was a PR that solved the problem of
+     attributes not sending events, but it should be included in the current
+     release (PR738).
+     - some issues with the Pool, but they will be further investigated before
+     reported them.
+     -  In one of the beamlines some user overwrite standard macros and
+     they endup with a MS without standard macros. The macros overwriting
+     the standard ones were created in a different module, in that case the
+     rest of the standard macros should not be affected. For avoinding that
+     this happens again they modified spock, so that it sends a message if
+     the standard macros are not overloaded.
+     
+   * DESY:
+     - in one beamline the measurement group stays for ever in MOVING at
+     some point of the scan. One has to stop the measurement group, calling
+     its Stop command from some other client, and the scan continues without
+     problems. A script was done for detecting this and solving it stoping
+     the measurement group. It happens several times per day. It will be
+     investigated in detail next week.
```

Comment:
Here we need to check if the measurement groups stays in MOVING and all its channels statys in MOVING as well. For example using Jive.